tests/fork_tests.move

```diff
@@ -172,3 +171,77 @@ fun cleanup_tests() {
     sui::test_utils::destroy(lc);
     scenario.end();
 }
+
+#[test]
+fun test_reorg() {
```

Comment:
Could you add a general comment to this test explaining the scenario in a very high level. You can say that we have chain [B1, B2...] At B4 we do fork [F5, F6] with higher difficulty....

Reply:
Done!